tests/test_bgp_af.rb

```diff
@@ -829,6 +833,6 @@ def test_table_map
     afs.each do |af|
       table_map(55, 'red', af)
     end
-    config('no route-policy sjc') if platform == :ios_xr
+    config_no_warn('no route-policy sjc') if platform == :ios_xr
```

Comment:
Out of curiosity, why does this generate a warning?  You created it on line 828, so you know it will exist, right?